packages/react-storage/src/components/StorageBrowser/context/actions/actions.tsx

```diff
@@ -26,9 +30,23 @@ export const DEFAULT_ACTIONS = {
   LIST_LOCATION_ITEMS: listLocationItemsAction,
   DOWNLOAD: downloadAction,
 };
+
+const INITIAL_DOWNLOAD_STATE: TransferTaskState = 'SUCCESS';
+
 export const INITIAL_VALUE = {
   LIST_LOCATION_ITEMS: { items: [], nextToken: undefined },
-  DOWNLOAD: {},
+  DOWNLOAD: {
+    result: Promise.resolve({
+      path: '',
+      body: {
+        blob: () => Promise.resolve(new Blob()),
+        json: () => Promise.resolve(null),
+        text: () => Promise.resolve(''),
+      },
+    }),
+    state: INITIAL_DOWNLOAD_STATE,
+    cancel: (): void => {},
+  },
```

Comment:
The initial state of an action should not have default values that point toward success or include async values. Let's take an approach that considers what information the UI cares abt from the result of calling `downloadAction`